bitnami/schema-registry/templates/statefulset.yaml

```
@@ -236,8 +236,13 @@ spec:
           {{- if .Values.lifecycleHooks }}
           lifecycle: {{- include "common.tplvalues.render" (dict "value" .Values.lifecycleHooks "context" $) | nindent 12 }}
           {{- end }}
-          {{- if or .Values.configuration .Values.existingConfigmap  .Values.log4j .Values.existingLog4jConfigMap (contains "SSL" $kafkaProtocol) .Values.extraVolumes }}
           volumeMounts:
+            - name: tmp-dir
+              mountPath: /tmp
+            - name: app-conf-dir
+              mountPath: /opt/bitnami/schema-registry/etc
+            - name: app-logs-dir
+              mountPath: /opt/bitnami/schema-registry/logs
```

Can we use a single emptyDir volume using subPaths?
```suggestion
            - name: empty-dir
              mountPath: /tmp
              subPath: tmp
            - name: empty-dir
              mountPath: /opt/bitnami/schema-registry/etc
              subPath: etc
            - name: empty-dir
              mountPath: /opt/bitnami/schema-registry/logs
              subPath: logs
```